debian/zookeeper/include/etc/confluent/docker/configure

```diff
@@ -25,8 +25,14 @@ dub ensure ZOOKEEPER_CLIENT_PORT
 dub path /etc/kafka/ writable
 
 # myid is required for clusters
+# Detect the ZOOKEEPER_SERVE_ID based on the hostname.
+# StatefulSets (kubernetes) are numbered from 0 so we have to always increment by 1
 if [[ -n "${ZOOKEEPER_SERVERS-}" ]]
 then
+  if [ -z "${ZOOKEEPER_SERVER_ID-unset}" ]; then
```

Comment:
Why are you using `if [ -z "${ZOOKEEPER_SERVER_ID-unset}"`?  It seems like that will never be true, since the `${FOO-bar}` construct gives the value "bar" if FOO is unset, right?